README.md

```diff
@@ -341,14 +239,15 @@ python3 -m generative_data_prep pipeline --input_file_path=./tests/examples/pret
 > [View decoded output](tests/examples/pretraining/decoded_data_prepped_pretraining.txt)
 
 
-### Generative tuning
-Generative tuning or "fine tuning" is a technique used to adapt a pre-trained language model to perform better at a specific task. This approach typically involves training the model on input data that is structured as a "prompt" followed by a "completion". The prompt represents the input for a specific task, while the completion is the output that the model should generate. During training, the model learns to generate the relevant completion tokens based on the context provided by the prompt tokens.
+### Fine-tuning
```

Comment:
Swith the order of pre-training and fine tuning? You pointed out most people will try fine tuning?